x-pack/plugins/threat_intelligence/public/mocks/test_providers.tsx

```diff
@@ -138,7 +138,7 @@ export const mockedServices = {
   },
 };
 
-export const TestProvidersComponent: FC<PropsWithChildren> = ({ children }) => (
+export const TestProvidersComponent: FC<PropsWithChildren<any>> = ({ children }) => (
```

Comment:
If left as `unknown` it would cause issues where `TestProvidersComponent` is used